Src/Support/Google.Apis.Auth.AspNetCore/Google.Apis.Auth.AspNetCore.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\CommonProjectProperties.xml" />
+
+  <PropertyGroup>
+    <!-- Only ASP.NET Core 2.0 on netstandard2.0 supported -->
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <!-- Beta release -->
+    <Version>$(Version)-beta01</Version>
+  </PropertyGroup>
+
+  <!-- nupkg information -->
```

Comment:
Not sure the comment is particularly useful - and is there any need for it to be in a separate property group? (I can see why other projects might look like this for historical reasons, but I wouldn't bother about that here.)

Reply:
This is the standard way csproj files are layed out in this repo. I'd rather keep to this standard if possible.